test/plugin/scenarios/mysql-scenario/src/main/resources/jdbc.properties

```diff
@@ -13,6 +13,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-mysql.url=jdbc:mysql://mysql-server:3306/test?serverTimezone=CST&enabledTLSProtocols=TLSv1.2&useSSL=false
+mysql.url=jdbc:mysql://mysql-server:3306/test?serverTimezone=CST&enabledTLSProtocols=TLSv1.2&useSSL=false&useServerPrepStmts=true
```

Comment:
Is this going to override the PreparedStatement cases?

Reply:
@xu1009 ☝️

Reply:
Generally, as this has been tested, I think we should be OK to revert the changes. To keep the duplicate cases for just this one thing seems a little overreactive.